db/migrate/20230816032645_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
```

Comment:
- [ ] As required for the project to o correctly match the the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) give to you in the project requirements, kindly ensure that the `author_id` column is used as the foreign_key of the `users` table. This was supposed to be an `author` column but of  type ``references`` to the the table `users`. i.e.
```ruby
 create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps

    t.references :author, foreign_key: { to_table: 'users' } 
  end
```